### PR TITLE
Enable get_molblock(details_json) from MinimalLib

### PR DESCRIPTION
**Pull request by ptosco**
CFFI's `get_molblock()` already supports `kekulize` and `includeStereo` parameters, but MInimalLib JS didn't; this PR corrects that.
I also exposed `useMolBlockWedging` and `addChiralHs` on `get-molblock()` in both libraries, as it is useful to make sure that copying the molecule to clipboard as molblock (for example) preserves exactly what is visible on screen if desired.

### CODE REVIEW

**Comment by ptosco** [comment]
The failure of the `Ubuntu_x64_limitexternaldependencies` build is unrelated to this PR.

**Comment by greglandrum** [comment]
> The failure of the `Ubuntu_x64_limitexternaldependencies` build is unrelated to this PR.

Yep, it's not a concern

File: Code/MinimalLib/common.h [hunk 1]
```diff
@@ -448,6 +449,30 @@ std::string process_rxn_details(
...
+    boost::property_tree::ptree pt;
+    std::istringstream ss;
+    ss.str(details_json);
+    boost::property_tree::read_json(ss, pt);
+    LPT_OPT_GET(includeStereo);
+    LPT_OPT_GET(kekulize);
+    LPT_OPT_GET(useMolBlockWedging);
+    LPT_OPT_GET(addChiralHs);
+  }
+  if (useMolBlockWedging) {
```
Comment (greglandrum) [inline Code/MinimalLib/common.h:467]: We should really create a version of `MolToMolBlock()` that (optionally) takes care of stuff like `reapplyMolBlockWedging()` and `addChiralHs()`

Comment (greglandrum) [reply Code/MinimalLib/common.h:467]: not for this PR, of course

File: Code/MinimalLib/common.h [hunk 2]
```diff
@@ -448,6 +449,30 @@ std::string process_rxn_details(
...
+    reapplyMolBlockWedging(mol);
+  }
+  if (addChiralHs) {
+    MolDraw2DUtils::prepareMolForDrawing(mol, false, true, false, false, false);
```
Comment (greglandrum) [inline Code/MinimalLib/common.h:471]: sigh... we should pull `addChiralHs()` out into a function in `MolOps.h` (again, not necessarily for this PR).

**Review by greglandrum — APPROVED** [review]
LGTM
The comments are more about things we should do in separate PRs

**Pull request merged**